wallet-webapps/src/main/webapp/vue-app/initComponents.js

```diff
@@ -18,6 +19,13 @@ const components = {
   'wallet-reward-transaction-history-chart-summary': TransactionHistoryChartSummary,
 };
 
+const externalComponentOptions = {
+  name: 'space-wallet-setting',
+  componentImpl: Vue.options.components['wallet-settings']
+};
+
+document.dispatchEvent(new CustomEvent('addExternalComponents', {'detail': externalComponentOptions}));
```

Comment:
Could you use a more precise name for the event ? addSpaceSettingsComponents

Reply:
done!